packages/nouns-contracts/tasks/create-proposal.ts

```diff
@@ -9,7 +9,7 @@ task('create-proposal', 'Create a governance proposal')
     types.string,
   )
   .setAction(async ({ nounsDaoProxy }, { ethers }) => {
-    const nounsDaoFactory = await ethers.getContractFactory('NounsDAOLogicV1');
+    const nounsDaoFactory = await ethers.getContractFactory('NounsDAOLogicV2');
```

Comment:
@eladmallel should this change back to V1 until we upgrade it?

Reply:
in practice I think this doesn't matter much, because the propose function signature is the same, but I think it's easy to change :)